edgexfoundry/device-sdk-c
# fix: Bypass JWT check in non-secure mode

fix: #474 







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-sdk-c/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions

Run EdgeX stack (non-secure mode)
Run device-random example https://github.com/edgexfoundry/device-sdk-c/tree/main/src/c/examples/random
Make an HTTP request to the device service: `curl -I -X GET 0:59999/api/v3/device/name/RandomDevice1/Switch`
Expect to receive a status code of 200

## New Dependency Instructions (If applicable)
